src/package.xml

```diff
@@ -196,6 +196,10 @@
         <members>svgIcon_EVT_Press</members>
         <name>AuraDefinitionBundle</name>
     </types>
+    <types>
+        <members>Case.Behavior Management</members>
```

Comment:
Similar to my question below - do we want this to be packaged or should it be unmanaged?

Reply:
ahhh yes the business process will likely want to be unmanaged as well just in case people don't want to use it, they can delete/user their own, etc.